config/default/core.extension.yml

```diff
@@ -30,7 +31,6 @@ module:
   datetime: 0
   datetime_range: 0
   dbal: 0
-  dblog: 0
```

Comment:
Would it be worth splitting this for ddev?

Reply:
Yes and I should copy the config split for Devel to Gov

Reply:
Done